src/azure-cli/azure/cli/command_modules/cdn/tests/latest/test_afd_route_scenarios.py

```diff
@@ -68,8 +68,8 @@ def test_afd_route_crud(self, resource_group):
                                           profile_name,
                                           custom_domain_name,
                                           host_name,
-                                          AfdCertificateType.managed_certificate,
-                                          AfdMinimumTlsVersion.tls12,
+                                          AfdCertificateType.managed_certificate.value,
+                                          AfdMinimumTlsVersion.tls12.value,
```

Comment:
Fix error when parse arg:
```
ERROR    cli.azure.cli.core.azclierror:azlogging.py:212 az afd custom-domain create: 'AfdCertificateType.MANAGED_CERTIFICATE' is not a valid value for '--certificate-type'. Allowed values: CustomerCertificate, ManagedCertificate, AzureFirstPartyManagedCertificate.
ERROR    az_command_data_logger:azlogging.py:213 az afd custom-domain create: 'AfdCertificateType.MANAGED_CERTIFICATE' is not a valid value for '--certificate-type'. Allowed values: CustomerCertificate, ManagedCertificate, AzureFirstPartyManagedCertificate.
```
We pass `AfdCertificateType.managed_certificate` as parameter, which implicitly convert it as string, but `Enum`'s `__str__` method changes in 3.11:
``` python
from enum import Enum

class Foo(str, Enum):
    BAR = "bar"

# Python 3.10
f"{Foo.BAR}"  # > bar

# Python 3.11
f"{Foo.BAR}"  # > Foo.BAR
```
`__str__` returns the classname.member (AfdCertificateType.MANAGED_CERTIFICATE) rather than member's value.
Add `.value` to get the real value of Enum.

Ref: 
https://github.com/python/cpython/issues/100458